frontend/public/locales/en/common.json

```diff
@@ -1,30 +1,31 @@
 {
   "Index": {
     "page_title": "Home | Beta.grants.gov",
-    "meta_description": "A one-stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities.",
-    "alert": "This website is a work in progress. To search for funding opportunities and apply, visit <LinkToGrants>www.grants.gov</LinkToGrants>.",
+    "meta_description": "A one‑stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities.",
+    "alert_title": "This website is a work in progress.",
+    "alert": "To search for funding opportunities and apply, visit <LinkToGrants>www.grants.gov</LinkToGrants>.",
     "goal_title": "What's the goal?",
     "goal_paragraph_1": "We want Grants.gov to be the simplest, most inclusive, and most gratifying tool ever built for posting, finding, sharing, and applying for financial assistance. Our mission is to increase access to grants and improve the grants experience for everyone.",
     "goal_title_2": "For applicants",
-    "goal_paragraph_2": "We’re building a one-stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities.",
+    "goal_paragraph_2": "We’re building a one‑stop shop for all federal discretionary funding to make it easy for you to discover, understand, and apply for opportunities.",
     "goal_title_3": "For grantmakers",
     "goal_paragraph_3": "If you work for a federal grantmaking agency, we’re making it easier for you to write, post, announce, and manage funding opportunities that reach your communities, including underserved communities.",
     "fo_title": "Improvements to funding opportunity announcements",
     "fo_paragraph_1": "Funding opportunities should not only be easy to find, share, and apply for. They should also be easy to read and understand. Our objective is to simplify and organize funding opportunities announcements. ",
     "fo_paragraph_2": "We want to help grantmakers write clear, concise announcements that encourage strong submissions from qualified applicants and make opportunities more accessible to everyone.",
     "fo_title_2": "View our grant announcement prototypes",
-    "fo_paragraph_3": "We recently simplified the language of four grant announcements and applied visual and user-centered design principles to increase their readability and usability.",
+    "fo_paragraph_3": "We recently simplified the language of four grant announcements and applied visual and user‑centered design principles to increase their readability and usability.",
```

Comment:
Using a non-breaking character in some of the hyphenated words so that they don't wrap to multiple lines.